### PR TITLE
updated README.md with reference genomes and alignment documentation

### PR DESCRIPTION
**Pull request by qiqing-li**
Some comments about the changes I made:
1. I added alignment.run_star_and_htseq to quick reference table to help users understand the paralleled relationship between alignment.10x_count and alignment.run_star_and_htseq, and the "subsidiary" relationship between alignment.run_star_and_htseq and aws_star. After I complete writing the script that wraps multiple 10x_count runs together, I'll edit the reference table again to leave only aws_star and say aws_10x_count there.
2. Please help check these two things about genome that I probably made mistakes: 1) In the subsection "How to launch alignment jobs", whether I used the proper genomes in the examples of running 10x_count, run_star_and_htseq, and aws_star; 2) In the subsection "How to choose the right genome (--taxon argument in running the alignment job)", whether I attached correct links for all those genomes.

Some thoughts about possible next steps. Some of them may be trivial and unnecessary, but if they sound good, I could work to fix them:
1. Make --s3_output_path positional in aws_star (it's now the only argument that's not positional), or make all the arguments not positional (which I think may be clearer).
2. aws_star uses --s3_input_path & 10x alignment uses --s3_input_dir. Could be a bit confusing. Could make both path or dir.
3. Delete --num_partitions and --partition_id arguments in run_star_and_htseq since the input folder is a single sample instead of a bunch of samples. This may help prevent confusion.
4. Update genomes to the latest versions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -259,3 +328,44 @@ By default this will share data in the `czb-seqbot` bucket. If you want to share
...
+ERROR:aegea:NoRegionError: You must specify a region.
+ERROR:aegea:Aegea will be unable to look up logs for old Batch jobs.
+The AWS CLI is not configured. Please configure it using instructions at http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html
+```
+To fix the error, go to the website for detailed instructions. Sample codes from the website is pasted below. Replace the sample values with your own values to configure the AWS CLI:
+
+```zsh
+(utilities-env) ➜ aws configure
+AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
```
Comment (jamestwebber) [inline README.md:349]: DO NOT COMMIT THIS

Comment (qiqing-li) [reply README.md]: just fixed!

**Comment by jamestwebber** [comment]
Only thing you might want to add is for the "PLUS" genomes. Besides transgenes, they also include sequences for ERCCs (these are a standardized mix of spike-in sequences used as a control).

**Comment by jamestwebber** [comment]
Why'd you close it?

**Pull request closed without merging**